daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/parsers/DelimiterParsers.scala

```diff
@@ -158,7 +158,7 @@ class DelimiterTextParser(
           return
         } else {
           // no match and no ES in delims
-          PE(start, "Delimiter not found.")
+          PE(start, "%s '%s' not found.", delimiterType.toString, start.mpstate.delimiters.last.lookingFor)
```

Comment:
This small change is a long time coming!

Reply:
Doesn't work great tho. Sometimes you see %NL; '%NL;' not found. Other times 'xy' where x and y are those little picture Unicode chars for newline.  In the runtime code there is not currently a way to go from the delimiter or set of delimiters being found, to the schema author's expression of those delimiters. This is needed for a good diagnostic.  I'll report this inconsistency as a bug.